lib/linter/linter.js

```diff
@@ -333,26 +334,42 @@ function createDisableDirectives({ type, value, justification, node }, ruleMappe
         // push to directives, if the rule is defined(including null, e.g. /*eslint enable*/)
         if (ruleId === null || !!ruleMapper(ruleId)) {
             if (type === "disable-next-line") {
+                const { line, column } = updateLocationInformation(
+                    {
+                        line: node.loc.end.line,
+                        column: node.loc.end.column
+                    },
```

Comment:
Can we simplify like this?
```suggestion
                const { line, column } = updateLocationInformation(
                    node.loc.end,
```

Reply:
Definitely. I'm not sure why it looked to me that it must be done this way.